README.md

```diff
@@ -11,7 +11,12 @@ Any pre-requisites that may not be covered by Ansible itself or the role should
 Role Variables
 --------------
 
-A description of the settable variables for this role should go here, including any variables that are in defaults/main.yml, vars/main.yml, and any variables that can/should be set via parameters to the role. Any variables that are read from other roles and/or the global scope (ie. hostvars, group vars, etc.) should be mentioned here as well.
+midonet_version: "midonet-5.2"
+midonet_release: "stable"
+openstack_version: "mitaka"
+plugin_release: "stable"
```

Comment:
do not hardcode MidoNet & OpenStack versions, as well as branches, but parametrize them all. Te above hard-coded values should only be a default values of these parameters.